packages/metadata-builders/tests/checksum-builder.spec.ts

```diff
@@ -58,7 +65,7 @@ describe("getChecksumBuilder properties", () => {
     )
   })
 
-  it("gives the same result regardless of entry point", () => {
+  it.skip("gives the same result regardless of entry point", () => {
```

Comment:
We should maintain this test, it kinda assures that all checksums are created properly. 👍🏻

Reply:
Sounds fair 👍🏻